### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Amen-Musingarimi**
**What I did** 

- [ ] Remove all hardcoded items from the tasks array.
- [ ] Create a new JavaScript file for the new functionality.
- [ ] Implement a function for adding a new task (add a new element to the array)
- [ ] Implement a function for deleting a task (remove an element from the array).
- [ ] All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/modules/editItem.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      }
+    });
+  });
+  item.addEventListener('keyup', (e) => {
+    if (e.key === 'Enter') {
+      array.forEach((task) => {
+        if (task.index === item.id) {
+          task.description = item.value;
+          localStorage.setItem('array', JSON.stringify(array));
+        }
```
Comment (iambenkis) [inline src/modules/editItem.js]: - [ ] I noticed that tasks are able to be edited or updated, but the changes made after editing are not save in the local storage :cry  . As part of CRUD requirements for this project, kindly ensure to create a function for editing tasks.

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hi @Amen-Musingarimi , 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Properly implemented function for adding and removing tasks

## Required Changes ♻️

_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/editItem.js (hunk 1 above)
Comment (Amen-Musingarimi) [reply src/modules/editItem.js]: @iambenkis Any idea how I can do it

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Amen-Musingarimi 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**